tests/unit/layers/models/layersManager.spec.ts

```diff
@@ -233,6 +318,25 @@ describe('layersManager', () => {
       expect(updateJsonMock).toHaveBeenCalledTimes(1);
     });
 
+    it('should successfully update layer + redis attributes', async () => {
+      // mock
+      const mockLayerName = 'redisExists-source';
+      const mockRedisLayerName = 'redisExists';
+
+      // action
+      expect.assertions(4);
+      const action = layersManager.updateLayer(mockLayerName, mockUpdateLayerRequest);
+
+      // expectation
+      await expect(action).toResolve();
+      const result = await MockConfigProvider.getJson();
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      expect(result.caches[mockLayerName].format).toBe(TileFormat.JPEG);
```

Comment:
you should assert it to be PNG before the update operation is called in the test